ui/imports/shared/status/StatusStickerPackClickPopup.qml

```diff
@@ -37,15 +37,17 @@ ModalPopup {
     property var stickers;
     signal buyClicked(string packId)
 
-    Component.onCompleted: {
+    onAboutToShow: {
+        stickersModule.getInstalledStickerPacks()
```

Comment:
Make sure the sticker (packs) are loaded even before the first time we open the regular Sticker popup; imho the NIM module should just lazily self initialize on first use, it shouldn't be necessary to call NIM methods from here